be/src/formats/parquet/group_reader.cpp

```diff
@@ -365,10 +365,8 @@ void GroupReader::_collect_field_io_range(const ParquetField& field, const TypeD
                                     active, ranges, end_offset);
         }
     } else if (field.type.type == LogicalType::TYPE_MAP) {
-        // ParquetFiled Map -> Map<Struct<key,value>>
-        DCHECK(field.children[0].type.type == TYPE_STRUCT);
         auto index = 0;
-        for (auto& child : field.children[0].children) {
+        for (auto& child : field.children) {
```

Comment:
Better to use 
```
for (int i = 0; i < field.children.size(); ++i)
```
BTW, the origin code is `field.children[0].children`, but this is `field.children`, which one is right?

Reply:
previous we will parse map as struct<map<key, value>>, now we parse map as map<key, value> directly.